### ISSUE TITLE
ktfmt fails to parse angle brackets in certain cases

### ISSUE THREAD

**Issue by myabc** [facebookincubator/ktfmt#51]
~~**Example 1.** In the case of a class inheriting from more than one class or interface, angle bracket syntax is used to denote the supertype from which the inherited implementation is taken.~~

```kotlin
package com.myandroidapp

import android.app.Application
import androidx.lifecycle.DefaultLifecycleObserver
import androidx.lifecycle.LifecycleOwner

class App : Application(), DefaultLifecycleObserver {

  override fun onCreate() {
    // com.google.googlejavaformat.FormattingError: 11:15: error: did not generate token "<"
    super<Application>.onCreate()
   // etc.
  }
}
```

~~_Expected behaviour:_ ktfmt parses/reformats the source code.~~
~~_Actual behaviour:_ results in an error similar to `com.google.googlejavaformat.FormattingError: 11:15: error: did not generate token "<"`~~

Update: appears to be fixed in 2c17040

**Example 2** Type Parameters in Annotations: for example, the `@TypeParceler` annotation from Kotlinx's Parcelize library allows you to explicitly specify an implementation of `Parcelable` for types that do not have built-in serialisation support. 

```kotlin
import java.util.UUID

import kotlinx.android.parcel.*

@Parcelize
@TypeParceler<UUID, UUIDParceler>()
data class User(
  val id: UUID
)
```

_Expected behaviour:_ ktfmt parses/reformats the source code.
_Actual behaviour:_ results in an error similar to `com.google.googlejavaformat.FormattingError: 5:21: error: expected token: '('; generated < instead` or `5:21: error: expected token: 'data'; generated < instead` if the empty parentheses are removed.

---

Using ktfmt 0.13 with Spotless plugin for Gradle 4.4.0; Gradle 6.5.0. Targeting Kotlin 1.3.72.

**Comment by myabc** [comment]
Since looked through the commit history. It appears this commit addresses the issue highlighted in the first example: https://github.com/facebookincubator/ktfmt/commit/2c17040aaced129d431fde5765e1c03d79e10ab0

**Comment by cgrushko** [comment]
@myabc thanks! if I understand correctly, it's a matter of releasing a new version?
(also, we should think about how to automate releases...)

**Comment by myabc** [comment]
@cgrushko I built ktfmt from source and installed it into my local Maven repository. I can confirm that 2c17040 does the trick: ktfmt no longer crashes on calls to `super` with angle bracket syntax to specify the supertype.

> I understand correctly, it's a matter of releasing a new version?
 
If only... 🙂

AFAICT the second issue I listed is still present on `master`. While it'd be great to see a release with the fix for multiple implementation inheritance, that release would not address all angle bracket syntax bugs.
